vuln/npm/490.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id": 490,
+  "title": "Denial of Service",
+  "overview": "Prototype pollution attack in node.extend",
+  "created_at": "2018-10-30",
+  "updated_at": "2019-01-03",
+  "publish_date": "2018-11-30",
+  "author": {
+    "name": "asgerf",
+    "website": null,
+    "username": "asgerf"
+  },
+  "module_name": "node.extend",
+  "cves": [
+    "CVE-2018-16491"
+  ],
+  "vulnerable_versions": "<2.0.1 || <1.1.7",
+  "patched_versions": "^1.1.7 || >= 2.0.1",
+  "recommendation": "Update node.extend module to version >=1.1.7 <2.0.1 || >= 2.0.1",
```

Comment:
The version range in here should probably updated too

Reply:
yes thanks Nick!